lua/README.md

```diff
@@ -0,0 +1,4 @@
+# About
+Contains compiled output from [fennel](../fnl) dir
```

Comment:
**issue**: Outdated information (we're not using fennel anymore)

**suggestion**: Update to `[src](../src)`